src/messages.cs

```diff
@@ -203,7 +203,7 @@ interface MSMessageTemplateLayout
 
 	[iOS (10,0)]
 	[BaseType (typeof(NSObject))]
-	interface MSSession {}
+	interface MSSession : NSSecureCoding {}
```

Comment:
@spouliot I tried running the intro tests on an iOS 10.0 sim and they all passed just fine.